plugins/modem-manager/fu-plugin-modem-manager.c

```diff
@@ -183,7 +187,7 @@ fu_plugin_mm_udev_uevent_cb (GUdevClient	*udev,
 static gboolean
 fu_plugin_mm_inhibit_device (FuPlugin *plugin, FuDevice *device, GError **error)
 {
-	static const gchar *subsystems[] = { "tty", "usbmisc", NULL };
+	static const gchar *subsystems[] = { "tty", "usbmisc", "wwan", NULL };
```

Comment:
does `wwan` only make sense when the new enough MM?  I just worry we might introduce a complexity of this matrix item:
new kernel, old MM, new fwupd.

Reply:
> does `wwan` only make sense when the new enough MM?

Yes, `wwan`  was supported in MM >= 1.17.1.

Reply:
Well, it will be supported in the next stable MM 1.16.6 release as well. This really depends on the kernel version in use; I don't think adding "wwan" to the list of subsystems is much of an issue.